src/wallet/rpcwallet.cpp

```diff
@@ -2886,6 +2879,11 @@ UniValue listtransactions(const UniValue& params, bool fHelp)
         }
     }
 
+    bool includeImmatureBTs = false;
+    if(params.size() > 5)
+        if(params[5].getBool())
```

Comment:
A minor change: it might be useful add isBool() check, otherwise a user can set '1' meaning 'True' and the cmd executes 'False'.
Or better: use get_bool() which throws an exception directly

Reply:
done